tests/protractor/pages/join.js

```diff
@@ -0,0 +1,97 @@
+var bedrock = GLOBAL.bedrock;
+
+var api = {};
+module.exports = api;
+
+var by = GLOBAL.by;
+var element = GLOBAL.element;
+var should = GLOBAL.should;
+var expect = GLOBAL.expect;
+
+api.getJoin = function() {
+  bedrock.get('/join');
+  return api;
+};
+
+api.navigateToHomePage = function() {
```

Comment:
Seems unusual for this to be in the `join` page API. It should be on a `home` page API and this page (or others) could access it as needed.